curriculum/superblock-structure/full-stack.json

```diff
@@ -664,6 +675,16 @@
         }
       ]
     },
+    {
+      "dashedName": "backend-javascript",
+      "comingSoon": true,
+      "modules": []
+    },
+    {
+      "dashedName": "python",
+      "comingSoon": true,
+      "modules": []
+    },
     {
       "chapterType": "exam",
```

Comment:
It really doesn't matter that much. It will stay hidden until we change `isUpcomingChange: false` on that block. I think it's fine as-is. The more important thing here is that it easily allows it to add any items as coming soon when we need to.

Reply:
That's a good point. Also, if you want to show it as coming soon, it's a trivial change. Add the flag and update intro.json and it'll pop into being.